src/app/content/components/Content.tsx

```diff
@@ -139,6 +140,7 @@ const Content: React.SFC = () => <Layout>
               <HideOverflowAndRedoPadding>
                 <Page />
               </HideOverflowAndRedoPadding>
+              <PrevNextBar />
```

Comment:
i think you probably want to move this up one line next to `Page` so it lines up perfectly with the text